packages/expo-updates/android/src/main/java/expo/modules/updates/loader/RemoteLoader.java

```diff
@@ -130,6 +130,14 @@ private void processManifest(Manifest manifest) {
 
     UpdateEntity newUpdateEntity = manifest.getUpdateEntity();
     UpdateEntity existingUpdateEntity = mDatabase.updateDao().loadUpdateWithId(newUpdateEntity.id);
+
+    // if something has gone wrong on the server and we have two updates with the same id
+    // but different scope keys, we should try to launch something rather than show a cryptic
+    // error to the user.
+    if (existingUpdateEntity != null && !existingUpdateEntity.scopeKey.equals(newUpdateEntity.scopeKey)) {
+      mDatabase.updateDao().setUpdateScopeKey(existingUpdateEntity, newUpdateEntity.scopeKey);
```

Comment:
Do you think it's worth logging in the event someone needs to diagnose issues on a device?